imagedephi/rules.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import TYPE_CHECKING
+
+import tifftools
+
+if TYPE_CHECKING:
+    from tifftools.tifftools import IFD, TagEntry
+
+
+@dataclass
+class TiffTagEntry:
+    tag: tifftools.TiffTag
+    entry: TagEntry
+
+
+class RuleType(Enum):
+    METADATA = 1
```

Comment:
```suggestion
    METADATA = "metadata"
```

Rather than arbitrary numbers, how about string values for all these `Enum`?

Reply:
Didn't notice you changed the recommendation to use `auto()`. Would that be preferred to the strings? I updated to strings in e5d83e6.